apps/src/templates/pane-header.module.scss

```diff
@@ -33,9 +51,18 @@
     background-color: #606060;
     color: #bfbfbf;
     border: none;
+
+    span {
+      color: #bfbfbf;
```

Comment:
Is there a color constant we can use for this?

Reply:
It's a specific color that we're only using in this exact file, so there're no constant for it